core/blockchain.go

```diff
@@ -213,27 +240,28 @@ func NewBlockChainDryrun(chainDb ethdb.Database, config *ChainConfig, pow pow.Po
 	return bc, nil
 }
 
-func (self *BlockChain) GetEventMux() *event.TypeMux {
-	return self.eventMux
+// GetEventMux returns the blockchain's event mux
+func (bc *BlockChain) GetEventMux() *event.TypeMux {
+	return bc.eventMux
 }
 
 // SetAtxi sets the db and in-use var for atx indexing.
-func (self *BlockChain) SetAtxi(a *AtxiT) {
-	self.atxi = a
+func (bc *BlockChain) SetAtxi(a *AtxiT) {
```

Comment:
Now here's a change I kind of regret that's included with this just because it's so pervasive and noopy. It's just a code style thing. But it should happen sooner or later, and makes integration with EF stuff a little smoother sometimes.